ui/src/App.jsx

```diff
@@ -41,13 +41,13 @@ const App = ({ signOut }) => {
   const [errors, setErrors] = useState([]);
   const [promptList, setPromptList] = useState([]);
   const [promptView, setPromptView] = useState({label: "Select a Prompt", value: "" });
-  const [promptText, setPromptText] = useState("")
+  const [promptText, setPromptText] = useState("");
 
 
   const submitQuery = async () => {
     try {
       setIsSubmitting(true);
-      const response = await API.post("api", "/", { body: { source, translation, llm, language, temperature }, headers: { "Content-Type": "text/plain" } });
+      const response = await API.post("api", "/evaluate-translation", { body: { source, translation, llm, language, temperature }, headers: { "Content-Type": "text/plain" } });
```

Comment:
Was there a reason for using `text/plain` over `application/json`?
`application/json` is probably more accurate.